doc/content/sfr/abtr_xsgen_workflow/abtr_xsgen_workflow.md

```diff
@@ -0,0 +1,332 @@
+  []
+[]
+
+[Executioner]
+  type = SweepUpdate
+  richardson_rel_tol = 1e-6
+  richardson_abs_tol = 1e-6
+  richardson_max_its = 100
+  richardson_value = eigenvalue
+  inner_solve_type = GMRes
+  max_inner_its = 4
+  inner_tol = 1.e-3
+  cmfd_acceleration = true
+[]
+
+[Materials]
+  [material_core_grid1]
+    type = MixedMatIDNeutronicsMaterial
+    isotopes = 'pseudo_REG'
+    densities = '1.0'
+    block = '100 300 400 700 800 900 1000'
+    grid_names = 'Tcool'
+    grid = '1'
+  []
+  [material_core_grid2]
+    type = MixedMatIDNeutronicsMaterial
+    isotopes = 'pseudo_REG'
+    densities = '1.0'
+    block = '200 500 600'
+    grid_names = 'Tcool Tfuel'
+    grid = '1 1'
+  []
+[]
+```
+
+Users are encouraged to look through this input file in detail and update input parameters accordingly based on their modeling needs. At a minimum however, a mesh file related to the heterogeneous reactor core needs to be defined with a name that matches `Mesh/eqv_core/input`, and boundary conditions for the full-core eigenvalue problem need to be defined by setting the appropriate outer boundary sidesets using `TransportSystems/VacuumBoundary` and `TransportSystems/ReflectingBoundary`. By looking at this file, it can be seen that the mapping between mesh blocks and cross section material ID is automatically done by calling `MixedMatIDNeutronicsMaterial` in the `[Materials]` block, and this assumes that the homogeneous input reactor core mesh has been generated with `EquivalentCoreMeshGenerator`.
+
+The heterogeneous reactor core mesh can be copied directly from the `xs_generation_mesh.i` (everything up to and including the `Mesh/het_core` block), and pasted into `core_hom_macro.i` above the `Mesh/eqv_core` block. The remainder of the `[Mesh]` block in `core_hom_macro.i` follows very similar steps as the ones explained in `xs_generation_mesh.i`, `EquivalentCoreMeshGenerator` is responsible for generating the homogeneous mesh from the input heterogeneous mesh specifications.
+
+In `core_hom_macro.i`, `EquivalentCoreMeshGenerator/target_geometry` is set to `full_hom` to specify a full-homogeneous reactor mesh as the output mesh. Additionally, `EquivalentCoreMeshGenerator/quad_center_elements` can be set to dictate whether the homogeneized hexagonal prism regions are discretized into 2 quadrilaterial prism regions (true) or 6 triangular prism regions (false), and `EquivalentCoreMeshGenerator/max_axial_mesh_size` sets the maximum axial mesh size to sub-discretize each axial layer of the output mesh.
+
+!media plots/eqv_fullhomcore.png
+       style=width:80%
+       id=fig:eqv_fullhomcore
+       caption=Generation of 3-D fully homogeneous core mesh (right) from 3-D heterogeneous core created through [!ac](RGMB) mesh generators (left).
+
+While `core_hom_macro.i` does not define a coarse mesh for the output mesh, this can be done manually by adding the following lines to the end of the `[Mesh]` block in `core_hom_macro.i`:
+
+```
+[Mesh]
```

Comment:
same

Reply:
@GiudGiud this is not part of any input file listing right now, since core_hom_macro.i does not define a coarse mesh. Should I add these parameters in the CLI section where I discuss how to run the problem?